src/Dfe.PlanTech.Web/Views/Recommendations/RecommendationsChecklist.cshtml

```diff
@@ -20,12 +20,12 @@
         <partial name="RecommendationPrintContent" model="@Model.AllContent"/>
     </div>
     <div class="govuk-grid-column-one-third print-buttons__container noprint">
-        <div id="recommendations-print-page" class="govuk-button govuk-button--inverse print-buttons__button">
+        <div id="recommendations-print-page" class="govuk-button govuk-button--inverse print-buttons__button js-only govuk-visually-hidden">
```

Comment:
do we need the aria hidden here too?

Reply:
Had to check but yes. I assumed the `govuk-visually-hidden` class would set `display:none` or `visibility:hidden` but this was not the case.

I've amended `js-only` to do the above, and we remove that using JS instead.

Same effect, simpler to implement.